root_orchestrator/system-manager-python/requirements.txt

```diff
@@ -18,4 +18,7 @@ werkzeug==2.0.3
 MarkupSafe~=2.1.1
 flask_swagger_ui
 mongomock==4.1.2
-git+https://github.com/oakestra/oakestra.git@develop#subdirectory=libraries/resource_abstractor_client
+#git+https://github.com/oakestra/oakestra.git@develop#subdirectory=libraries/resource_abstractor_client
```

Comment:
@Malyuk-A change the library dependencies to be pointing to the devel branch before merging

Reply:
If they manually point to develop, how do we make sure that stable branch points to main?

Reply:
That is a great question.

This is currently already used by the Resource Abstractor Client Library. (@melkodary )
I currently see two solutions:
1) Having a dedicated MR for migrating things from devel to stable - in this MR the links need to be adjusted, this is not too pretty.
2) Publish these libraries as pip packages with proper versioning, same redundancy - "headaches" follow. 

I recommend discussing this in a maintainers meeting to come up with a nice convention.

Reply:
I believe the correct approach is the second solution proposed. However for the time being I think there is another solution that could be implemented - create the PR for library and merge it before merging the stuff of the main PR. This way no need to change the links everytime. In other words create two PRs: 
1. one for the new library or modification to the library (assuming changes don't break anything) and merge it.
2. Next create the main PR that uses this library.